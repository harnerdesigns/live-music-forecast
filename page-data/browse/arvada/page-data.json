{"componentChunkName":"component---src-templates-city-jsx","path":"/browse/arvada/","result":{"data":{"events":{"totalCount":0,"edges":[]},"venues":{"totalCount":0,"edges":[]}},"pageContext":{"city":"Arvada","today":"2022-02-25T12:15:40.017Z","dateFormat":"MM/DD/YYYY"}},"staticQueryHashes":[]}