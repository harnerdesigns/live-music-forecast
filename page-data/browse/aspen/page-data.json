{"componentChunkName":"component---src-templates-city-jsx","path":"/browse/aspen/","result":{"data":{"events":{"totalCount":0,"edges":[]},"venues":{"totalCount":0,"edges":[]}},"pageContext":{"city":"Aspen","today":"2022-05-17T00:45:55.357Z","dateFormat":"MM/DD/YYYY"}},"staticQueryHashes":[]}