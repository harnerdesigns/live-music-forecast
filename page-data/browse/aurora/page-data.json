{"componentChunkName":"component---src-templates-city-jsx","path":"/browse/aurora/","result":{"data":{"events":{"totalCount":4,"edges":[{"node":{"fields":{"slug":"/event/ladies-night-10-27-2021"},"data":{"StartDate":"2021-10-28T00:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Ladies Night!","Description":"See you every Wednesday night for line dancing and more!$5 door, Dance Lessons from 6:30-7:30 then open dancing until 2am","SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":null,"Image":[{"url":"https://dl.airtable.com/.attachments/5b82a4685dd9e226b9b042b5f66b3c9e/36747835/c58dd0af-cda8-49b2-bdaf-068540c3a677_106541_TABLET_LANDSCAPE_LARGE_16_9.jpg"}],"Venue_City":["Aurora"],"Venue_Name":["Stampede"],"Venues":[{"fields":{"slug":"/venues/stampede"},"data":{"City":"Aurora","Name":"Stampede"}}]}}},{"node":{"fields":{"slug":"/event/los-pescadores-del-rio-conchos-ulices-chaidez-la-nueva-corriente-10-30-2021"},"data":{"StartDate":"2021-10-31T03:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Los Pescadores del Rio Conchos, Ulices Chaidez, La Nueva Corriente","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":null,"Image":[{"url":"https://dl.airtable.com/.attachments/9b33026d0413002267fd32f9a16ac513/0f75f0f6/5fefbd1c-973b-4b0e-9b2e-d78e4ce37548_106111_TABLET_LANDSCAPE_LARGE_16_9.jpg"}],"Venue_City":["Aurora"],"Venue_Name":["Stampede"],"Venues":[{"fields":{"slug":"/venues/stampede"},"data":{"City":"Aurora","Name":"Stampede"}}]}}},{"node":{"fields":{"slug":"/event/ladies-night-11-10-2021"},"data":{"StartDate":"2021-11-11T01:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Ladies Night!","Description":"See you every Wednesday night for line dancing and more!$5 door, Dance Lessons from 6:30-7:30 then open dancing until 2am","SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":null,"Image":[{"url":"https://dl.airtable.com/.attachments/6840d466b46a3b947ca5a115cd7e99c4/3e2ede36/c58dd0af-cda8-49b2-bdaf-068540c3a677_106541_TABLET_LANDSCAPE_LARGE_16_9.jpg"}],"Venue_City":["Aurora"],"Venue_Name":["Stampede"],"Venues":[{"fields":{"slug":"/venues/stampede"},"data":{"City":"Aurora","Name":"Stampede"}}]}}},{"node":{"fields":{"slug":"/event/liberacion-industria-del-amor-samuray-11-27-2021"},"data":{"StartDate":"2021-11-28T04:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Liberacion, Industria Del Amor, Samuray","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":null,"Image":[{"url":"https://dl.airtable.com/.attachments/9b33026d0413002267fd32f9a16ac513/0f75f0f6/5fefbd1c-973b-4b0e-9b2e-d78e4ce37548_106111_TABLET_LANDSCAPE_LARGE_16_9.jpg"}],"Venue_City":["Aurora"],"Venue_Name":["Stampede"],"Venues":[{"fields":{"slug":"/venues/stampede"},"data":{"City":"Aurora","Name":"Stampede"}}]}}}]},"venues":{"totalCount":1,"edges":[{"node":{"fields":{"slug":"/venues/stampede"},"data":{"Name":"Stampede","Description":null,"Tags":null,"Featured":null,"City":"Aurora","AccentColor":"#ffffff","Logo":[{"url":"https://dl.airtable.com/.attachments/b1ea82e201a8e31dea1fb35dd68ab0c5/673c62ed/6953v.jpg"}]}}}]}},"pageContext":{"city":"Aurora","today":"2021-10-26T17:37:20.904Z","dateFormat":"MM/DD/YYYY"}},"staticQueryHashes":[]}