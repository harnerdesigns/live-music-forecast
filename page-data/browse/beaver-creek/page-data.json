{"componentChunkName":"component---src-templates-city-jsx","path":"/browse/beaver-creek/","result":{"data":{"events":{"totalCount":1,"edges":[{"node":{"fields":{"slug":"/event/kenny-g-07-11-2022"},"data":{"StartDate":"2022-07-12T01:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Kenny G","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":[{"fields":{"slug":"/artist/kenny-g"},"data":{"Name":"Kenny G"}}],"Image":[{"url":"https://dl.airtable.com/.attachments/a5bb9f6cc35314cc2e84b07e5ce2e79b/cfc8d627/120de5c3-5da8-4708-aa63-3319aa178388_1627401_EVENT_DETAIL_PAGE_16_9.jpg?ts=1652595256&userId=usr6NqVXGs7Gt4dhJ&cs=4b640ff42392edf5"}],"Venue_City":["Beaver Creek"],"Venue_Name":["Vilar Performing Arts Center"],"Venues":[{"fields":{"slug":"/venues/vilar-performing-arts-center"},"data":{"City":"Beaver Creek","Name":"Vilar Performing Arts Center"}}]}}}]},"venues":{"totalCount":1,"edges":[{"node":{"fields":{"slug":"/venues/vilar-performing-arts-center"},"data":{"Name":"Vilar Performing Arts Center","Description":null,"Tags":null,"Featured":null,"City":"Beaver Creek","AccentColor":null,"Logo":null}}}]}},"pageContext":{"city":"Beaver Creek","today":"2022-05-15T06:14:28.646Z","dateFormat":"MM/DD/YYYY"}},"staticQueryHashes":[]}