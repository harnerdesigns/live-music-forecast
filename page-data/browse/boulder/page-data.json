{"componentChunkName":"component---src-templates-city-jsx","path":"/browse/boulder/","result":{"data":{"events":{"totalCount":2,"edges":[{"node":{"fields":{"slug":"/event/kansas-05-15-2022"},"data":{"StartDate":"2022-05-16T01:30:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Kansas","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":[{"fields":{"slug":"/artist/kansas"},"data":{"Name":"Kansas"}}],"Image":[{"url":"https://dl.airtable.com/.attachments/a2b1744a4327eb4c5d652a9af322e82c/6b3f85f7/2164f073-79d2-424a-ae0b-2f2c0554fd06_1487791_TABLET_LANDSCAPE_LARGE_16_9.jpg?ts=1650415596&userId=usr6NqVXGs7Gt4dhJ&cs=3e940363e2e6c36b"}],"Venue_City":["Boulder"],"Venue_Name":["Macky Auditorium"],"Venues":[{"fields":{"slug":"/venues/macky-auditorium"},"data":{"City":"Boulder","Name":"Macky Auditorium"}}]}}},{"node":{"fields":{"slug":"/event/kenny-g-07-09-2022"},"data":{"StartDate":"2022-07-10T01:30:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Kenny G","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":[{"fields":{"slug":"/artist/kenny-g"},"data":{"Name":"Kenny G"}}],"Image":[{"url":"https://dl.airtable.com/.attachments/6bd658213785ee52c4d214058eb4c529/24596798/b211e971-108b-44c1-b91e-095601546deb_1559042_TABLET_LANDSCAPE_LARGE_16_9.jpg?ts=1650415596&userId=usr6NqVXGs7Gt4dhJ&cs=c9929ee95d7925f6"}],"Venue_City":["Boulder"],"Venue_Name":["Chautauqua Auditorium"],"Venues":[{"fields":{"slug":"/venues/chautauqua-auditorium"},"data":{"City":"Boulder","Name":"Chautauqua Auditorium"}}]}}}]},"venues":{"totalCount":2,"edges":[{"node":{"fields":{"slug":"/venues/chautauqua-auditorium"},"data":{"Name":"Chautauqua Auditorium","Description":null,"Tags":null,"Featured":null,"City":"Boulder","AccentColor":"#ffffff","Logo":[{"url":"https://dl.airtable.com/.attachments/2517517dedf61890af17dea22f1bb914/64b66ace/6805v.gif?ts=1650415596&userId=usr6NqVXGs7Gt4dhJ&cs=05bf6feaf2d37096"}]}}},{"node":{"fields":{"slug":"/venues/macky-auditorium"},"data":{"Name":"Macky Auditorium","Description":null,"Tags":null,"Featured":null,"City":"Boulder","AccentColor":"#000000","Logo":[{"url":"https://dl.airtable.com/.attachments/49b7e788bad9e6caa8106231afc0be86/8f1b6ec8/6758v.jpg?ts=1650415596&userId=usr6NqVXGs7Gt4dhJ&cs=d818baa205560fd4"}]}}}]}},"pageContext":{"city":"Boulder","today":"2022-04-20T00:46:44.098Z","dateFormat":"MM/DD/YYYY"}},"staticQueryHashes":[]}