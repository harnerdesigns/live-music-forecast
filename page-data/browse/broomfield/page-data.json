{"componentChunkName":"component---src-templates-city-jsx","path":"/browse/broomfield/","result":{"data":{"events":{"totalCount":4,"edges":[{"node":{"fields":{"slug":"/event/alt-j-w-portugal-the-man-03-23-2022"},"data":{"StartDate":"2022-03-24T01:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Alt-J w/ Portugal the Man","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":null,"Image":[{"url":"https://dl.airtable.com/.attachments/d057cdb1ed0fe281df60a3888b927c99/65f1aaa5/b794a3dd-88a2-4b5e-8aa3-1646f906f1db_1537501_RETINA_PORTRAIT_3_2.jpg"}],"Venue_City":["Broomfield"],"Venue_Name":["1STBANK Center"],"Venues":[{"fields":{"slug":"/venues/1-stbank-center"},"data":{"City":"Broomfield","Name":"1STBANK Center"}}]}}},{"node":{"fields":{"slug":"/event/jack-white-the-supply-chain-issues-tour-06-11-2022"},"data":{"StartDate":"2022-06-12T02:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"JACK WHITE: The Supply Chain Issues Tour","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":["Modern Rock","Blues Rock","Rock","Alternative Rock","Indie Rock","Modern Blues Rock"],"Artists":[{"fields":{"slug":"/artist/jack-white"},"data":{"Name":"Jack White"}}],"Image":[{"url":"https://dl.airtable.com/.attachments/dc360acf71a73d48afbb1c03f9209f9a/23fb570c/e194ab1a-3952-405c-b44e-54c9a5da4c6c_1579201_TABLET_LANDSCAPE_3_2.jpg"}],"Venue_City":["Broomfield"],"Venue_Name":["1STBANK Center"],"Venues":[{"fields":{"slug":"/venues/1-stbank-center"},"data":{"City":"Broomfield","Name":"1STBANK Center"}}]}}},{"node":{"fields":{"slug":"/event/jack-white-vip-upgrades-06-11-2022"},"data":{"StartDate":"2022-06-12T02:00:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Jack White - VIP Upgrades","Description":null,"SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":["Modern Rock","Blues Rock","Rock","Alternative Rock","Indie Rock","Modern Blues Rock"],"Artists":[{"fields":{"slug":"/artist/jack-white"},"data":{"Name":"Jack White"}}],"Image":[{"url":"https://dl.airtable.com/.attachments/dc360acf71a73d48afbb1c03f9209f9a/23fb570c/e194ab1a-3952-405c-b44e-54c9a5da4c6c_1579201_TABLET_LANDSCAPE_3_2.jpg"}],"Venue_City":["Broomfield"],"Venue_Name":["1STBANK Center"],"Venues":[{"fields":{"slug":"/venues/1-stbank-center"},"data":{"City":"Broomfield","Name":"1STBANK Center"}}]}}},{"node":{"fields":{"slug":"/event/moved-venues-parkway-drive-05-26-2023"},"data":{"StartDate":"2023-05-27T00:45:00.000Z","EndDate":null,"DoorsTime":null,"Name":"Moved Venues: Parkway Drive","Description":"This event has moved to Mission Ballroom, 4242 Wynkoop St. Denver, CO. and has now been Postponed/new date TBD.","SoldOut":null,"Tags":null,"Featured":null,"Artist_Genres":null,"Artists":[{"fields":{"slug":"/artist/parkway-drive"},"data":{"Name":"Parkway Drive"}}],"Image":[{"url":"https://dl.airtable.com/.attachments/cf64a13328a5fbbb98f091dc59c3471b/3d1f14b3/806fd505-b871-403a-9838-a53d0a8ac3d4_1553901_RECOMENDATION_16_9.jpg"}],"Venue_City":["Broomfield"],"Venue_Name":["1STBANK Center"],"Venues":[{"fields":{"slug":"/venues/1-stbank-center"},"data":{"City":"Broomfield","Name":"1STBANK Center"}}]}}}]},"venues":{"totalCount":1,"edges":[{"node":{"fields":{"slug":"/venues/1-stbank-center"},"data":{"Name":"1STBANK Center","Description":null,"Tags":null,"Featured":null,"City":"Broomfield","AccentColor":"#ffffff","Logo":null}}}]}},"pageContext":{"city":"Broomfield","today":"2022-03-21T18:12:20.171Z","dateFormat":"MM/DD/YYYY"}},"staticQueryHashes":[]}