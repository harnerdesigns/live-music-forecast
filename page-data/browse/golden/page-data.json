{"componentChunkName":"component---src-templates-city-jsx","path":"/browse/golden/","result":{"data":{"events":{"totalCount":0,"edges":[]},"venues":{"totalCount":0,"edges":[]}},"pageContext":{"city":"Golden","today":"2022-03-11T00:39:45.350Z","dateFormat":"MM/DD/YYYY"}},"staticQueryHashes":[]}